MODULE.bazel

```diff
@@ -1,3 +1,9 @@
+"""Buildfarm build and test"""
+
+# NOTE: When editing this file, also update the lockfile.
+#   bazel mod deps --lockfile_mode=update
+# See also: https://bazel.build/external/lockfile
+
```

Comment:
Should we then remove this line? Don't really see why we need any modification to this file.